modules/database/src/ioc/dbStatic/dbStaticLib.c

```diff
@@ -2380,7 +2380,17 @@ long dbParseLink(const char *str, short ftype, dbLinkInfo *pinfo)
         /* filter modifiers based on link type */
         switch(ftype) {
         case DBF_INLINK: /* accept all */ break;
-        case DBF_OUTLINK: pinfo->modifiers &= ~pvlOptCPP; break;
+        case DBF_OUTLINK:
+            if(pinfo->modifiers & (pvlOptCPP|pvlOptCP)){
+                fprintf(stderr, "Warning: dbParseLink: Output link to %s was specified with \n\
```

Comment:
I think this should be switched to `errlogPrintf()`.  Generally we avoid synchronous logging when it can happen after `iocInit()`.